src/components/Comments.jsx

```diff
@@ -19,13 +23,66 @@ const Comments = ({ article_id, comments, setComments }) => {
       });
   }, [article_id, setComments]);
 
+  const handleCommentSubmit = (event) => {
+    event.preventDefault();
+    setIsLoading(true);
+    const newComment = {
+      article_id: article_id,
+      username: username,
+      body: commentBody,
+    };
+    postComment(newComment)
+      .then((response) => {
+        setHasPosted(true);
+        setComments([response.data.comment, ...comments]);
+        setIsLoading(false);
+      })
+      .catch((err) => {
+        setIsError(err);
+      });
+  };
+
   if (isError)
     return <p>Sorry, we are unable to load comments at the moment</p>;
+
   if (isLoading) return <p>Loading comments...</p>;
 
   return (
     <section className="Comments">
       <h3>Comments</h3>
+      <form onSubmit={handleCommentSubmit} className="postCommentForm">
+        <label htmlFor="comment-input-box">
+          post your comment in the box below!
+        </label>
+        <textarea
+          required
+          type="text"
+          value={commentBody}
+          placeholder="Write your comment here..."
+          onChange={(event) => {
+            setCommentBody(event.target.value);
+          }}
+        />
+        <label htmlFor="username-input-box">What is your username?</label>
+        <select
```

Comment:
Username should be pulled from the logged-in user, rather than having to ask the user to fill it in every time. You can either add some fake log-in functionality, or hard-code a permanent user based on the users in the database